lib/smart_answer_flows/energy-grants-calculator.rb

```diff
@@ -470,6 +470,9 @@ def define
         precalculate :flat_type do
           flat_type
         end
+        precalculate :under_green_deal do
```

Comment:
@floehopper I don't have access to /that/[this](https://trello.com/c/Ldn6KJeE) Trello board